module.txt

```diff
@@ -14,6 +14,7 @@
         { "id": "CoreRendering", "minVersion": "1.0.0" },
         { "id": "CoreWorlds", "minVersion": "1.1.0", "maxVersion": "3.0.0" },
         { "id": "EdibleSubstance", "minVersion": "1.0.0" },
+        { "id": "WildAnimals", "minVersion": "1.1.0" },
```

Comment:
According to the build logs we never published v1.1.0 of WildAnimals, and the latest available version is 1.1.0-SNAPSHOT.

Reply:
Not sure how versioning works exactly. I declared a new event in WildAnimals, so I thought I can't use the -SNAPSHOT version. Does the SNAPSHOT version change according to the latest build of WildAnimals?

Reply:
Yes, SNAPSHOT builds override each other. That's nice for testing and iterations, but comes with its own set of problems when using SNAPSHOT releases in "productive" code. 

We usually aim to have proper releases of all modules before releasing a new version of the game, but during development we work with SNAPSHOTs a lot (not necessarily a best practice, but that's what we do).